learningresources/search_indexes.py

```diff
@@ -0,0 +1,26 @@
+"""
+Index information for Haystack.
+
+http://django-haystack.readthedocs.org/en/latest/tutorial.html
+"""
+from haystack.indexes \
+    import SearchIndex, Indexable, CharField  # pragma: no cover
```

Comment:
>  As a config file it doesn't get covered at all.

I'm confused by this, If the config file is never imported by python, than it isn't being used, and shouldn't be included... For example both `cas/conf.py` and `lore/settings.py` are covered almost entirely simply by being there, since config python files still show as covered when the project starts up...  Won't this show as covered as soon as we have a view or something that uses haystack? Or a test for a signal or something?  What happens if you start up `coverage run ipython` and import and do some stuff with Haystack?  It should show this as covered I would think.